corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -412,6 +447,8 @@ def _row_to_row_elem(deferred_fields, filter_options_by_field, row, index, is_to
         rows, filters_elem = generate_rows_and_filters(
             self.report_data_cache, report_config, restore_user, _row_to_row_elem
         )
+        # the v2 provider writes each report to its own fixture so we only care about the max row_count
+        self.row_count = max(len(rows), self.row_count)
```

Comment:
Why not just `len(rows)`? I'm missing the reason for using `max` here.

Reply:
Answering my own question: I was confused because I thought there was one `ReportFixturesProviderV2` for each fixture, but I see there is only one per restore.